### PR TITLE
Merge dev into demo

### PR DESCRIPTION
**Pull request by MatanYadaev**

### CODE REVIEW

File: apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx [hunk 1]
```diff
@@ -91,6 +91,13 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
         const properties = Object.keys(document?.propertiesSchema?.properties ?? {}).reduce(
           (acc, curr) => {
             if (!formData?.[curr]) return acc;
+
+            if (document?.propertiesSchema?.properties?.[curr]?.format === 'date-time') {
```
Comment (alonp99) [inline apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx]: Can you leave a comment explaining this fix?

File: apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx [hunk 2]
```diff
@@ -91,6 +91,13 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
...
+              if (formData?.[curr]?.length === 16) {
+                formData[curr] = `${formData[curr]}:00`;
+              }
+            }
+
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx]:
```suggestion
            if (document?.propertiesSchema?.properties?.[curr]?.format === 'date-time' && formData?.[curr]?.length === 16) {
                formData[curr] = `${formData[curr]}:00`;
            }
```

Comment (Omri-Levy) [reply apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx]: `formData` here is React state. Mutating it is a bad practice and may behave in unexpected ways.

File: apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx (hunk 1 above)
Comment (MatanYadaev) [reply apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx]: Yes, sorry: HTML5 `datetime-local` input holds a date time value. It should return `YYYY-MM-DDTHH:MM:SS`, but there is an edge-case when the seconds are `00`, it returns `YYYY-MM-DDTHH:MM` instead.
So here I fixed it, ensuring that when there are no seconds, I add `:00` to the string: `YYYY-MM-DDTHH:MM:00`

**Review by Omri-Levy — APPROVED** [review]

**Pull request merged**